arc/conformance/plugin/conformance.yaml

```diff
@@ -0,0 +1,15 @@
+sonobuoy-config:
+  driver: Job
+  plugin-name: akv-secrets-provider-arc-conformance
+  result-format: junit
+spec:
+  image: k8supstream.azurecr.io/oss/azure/secrets-store/provider-azure-arc-conformance:v1.0.1
```

Comment:
Isn't the staging image `upstream.azurecr.io/oss/azure/secrets-store/provider-azure-arc-conformance`? 

Provider image for ref:
```bash
➜ docker buildx imagetools inspect upstream.azurecr.io/oss/azure/secrets-store/provider-azure:v1.0.0
Name:      upstream.azurecr.io/oss/azure/secrets-store/provider-azure:v1.0.0
MediaType: application/vnd.docker.distribution.manifest.list.v2+json
Digest:    sha256:3f8b0c5f50b7ce8e8fc696852c37c4bc303713a272eb55ff8c83d8ecef97bde1
```

```
➜ docker buildx imagetools inspect k8upstream.azurecr.io/oss/azure/secrets-store/provider-azure:v1.0.0
INFO[0000] trying next host                              error="failed to do request: Head \"https://k8upstream.azurecr.io/v2/oss/azure/secrets-store/provider-azure/manifests/v1.0.0\": dial tcp: lookup k8upstream.azurecr.io: no such host" host=k8upstream.azurecr.io
```

Reply:
This is what I was confirming earlier. Isn't staging `k8supstream.azurecr.io/oss/azure/secrets-store`? and prod one `k8supstream.azurecr.io/public/oss/azure/secrets-store` with only difference of `public`?

Also I tried simple docker pull  `upstream.azurecr.io` and seems like it's not publicly accessible.

```
$ docker pull upstream.azurecr.io/oss/azure/secrets-store/provider-azure-arc-conformance:v1.0.1
Error response from daemon: Head https://upstream.azurecr.io/v2/oss/azure/secrets-store/provider-azure-arc-conformance/manifests/v1.0.1: unauthorized: authentication required

Reply:
@sozercan Is the staging registry public? Or do you recommend we use mcr for conformance images? I see osm is using their own registry.

Reply:
Synced with @sozercan offline. Let's use MCR for image.

Reply:
sounds good!